Closing Connection Section (#239)
dunglas/mercure: docs/getting-started.md

The `EventSource` class is available [in all modern web browsers](https://caniuse.com/#feat=eventsource). And for legacy browsers, [there are polyfills](ecosystem/awesome.md#useful-related-libraries).

## Closing Connection

It is important to close this connection between the client and the hub if it is no longer needed.

> Opened connections have continous buffer that will be drain your application resources.

Reviewer: ```suggestion
Opened connections have a continuous buffer that will drain your application resources. 
```

> This is especially true when using Single Page Applications based on ReactJS for example: the connection is maintained when the component that created it is unmounted.

Reviewer: ```suggestion
This is especially true when using Single Page Applications (based on e.g. ReactJS): the connection is maintained even if the component that created it is unmounted.
```
